NitroxClient/GameLogic/DevConsoleSpawnEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.Helper;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class DevConsoleSpawnEvent
+    {
+        private readonly IPacketSender packetSender;
+        private readonly Vehicles vehicles;
+
+        public DevConsoleSpawnEvent(IPacketSender packetSender, Vehicles vehicles)
+        {
+            this.packetSender = packetSender;
+            this.vehicles = vehicles;
+        }
+
+        /// <summary>
+        /// List of things that can be spawned : https://subnauticacommands.com/items
+        /// Vehicle that can be spawn : Seamoth, Exosuit
+        /// </summary>
+        public void Spawn(GameObject gameObject)
```

Comment:
We will likely want a different handler (maybe just method now) for each of the various spawn types.